metrics/scaling/bb.json.in

```diff
@@ -28,11 +28,7 @@
                 "containers": [{
                     "name": "bb",
                     "image": "busybox",
-                    "command": [
```

Comment:
At first I wondered does it matter if there is busybox tail reading /dev/null or busybox netcat reading a socket without getting connections. But indeed, there is a difference: `tail -f /dev/null` process wakes up every second, while `nc` does nothing until it gets a connection. And, of course, while the number of open file descriptors is the same, nc holds a socket type file, tail does not.

To remedy this, I added a concept of "workload type", because it helps responding to this and the later concern on `json` in the future as well. Now the current default workload is implemented by `workloads/bb_workload_default.cmd`, while netcat-based latency workloads are in `workloads/bb_workloads_*.nc`.